src/lib/rest/rest.cpp

```diff
@@ -719,6 +744,27 @@ std::string defaultServicePath(const char* url, const char* method)
 
 /* ****************************************************************************
 *
+* apiVersionGet - 
+*
+* This function returns the version of the API for the incoming message,
+* based on the URL.
+* If the URL starts with "/v2" then the request is considered API version 2.
+* Otherwise, API version 1.
+*/
+static std::string apiVersionGet(const char* path)
+{
+  if ((path[1] == 'v') && (path[2] == '2'))
```

Comment:
This way of doing it is *many* times faster than to call strcmp.
It is true that when we get up to version 20 one more check is needed (path[3] == '/').
I don't think we will ever get there ...